### PR TITLE
Version lock u-root and harvey-os Go repos

### PR DESCRIPTION
**Pull request by rminnich**
This is so we can avoid all this go.{mod,sum} jitter.

Signed-off-by: Ronald G Minnich <rminnich@gmail.com>

### CODE REVIEW

**Review by sevki — APPROVED** [review]

File: bootstrap.sh [hunk 1]
```diff
@@ -9,10 +9,10 @@ echo Building the build tool...
 GO111MODULE=on go get ./util/src/harvey/cmd/...
 
 echo Fetching u-root and building it...
-GO111MODULE=on go get github.com/u-root/u-root
+GO111MODULE=on go get github.com/u-root/u-root@c6181ee270c59d542825e89a7c02c8c25b30373b
```
Comment (fhs) [inline bootstrap.sh:12]: This fails on Linux:
```
# github.com/u-root/u-root/pkg/ulog
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:92:48: undefined: unix.SYSLOG_ACTION_CONSOLE_LEVEL
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:105:25: undefined: unix.SYSLOG_ACTION_CLEAR
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:111:22: undefined: unix.SYSLOG_ACTION_READ_ALL
/home/fhs/go/pkg/mod/github.com/u-root/u-root@v6.0.1-0.20200822172051-c6181ee270c5+incompatible/pkg/ulog/klog_linux.go:116:22: undefined: unix.SYSLOG_ACTION_READ_CLEAR
```
I had to add `golang.org/x/sys` (with the version listed in u-root `Gopkg.lock`) as an additional argument: 
```suggestion
GO111MODULE=on go get github.com/u-root/u-root@c6181ee270c59d542825e89a7c02c8c25b30373b golang.org/x/sys@05986578812163b26672dabd9b425240ae2bb0ad
```

File: bootstrap.sh [hunk 2]
```diff
@@ -9,10 +9,10 @@ echo Building the build tool...
...
 
 echo Fetch harvey-os.org commands and build them into $GOBIN
-GO111MODULE=on go get harvey-os.org/cmd/...
+GO111MODULE=on go get harvey-os.org@bcfa7228c80c35fc627c077fdd5918db2181489e
```
Comment (fhs) [inline bootstrap.sh:15]: Why is `/cmd/...` remove? There is no Go package at the top-level directory.
```suggestion
GO111MODULE=on go get harvey-os.org/cmd/...@bcfa7228c80c35fc627c077fdd5918db2181489e
```

File: go.mod [hunk 3]
```diff
@@ -5,8 +5,8 @@ go 1.13
 require (
 	github.com/google/goexpect v0.0.0-20200703111054-623d5ca06f56 // indirect
 	github.com/google/goterm v0.0.0-20190703233501-fc88cf888a3f // indirect
-	github.com/u-root/u-root v6.0.0+incompatible
-	golang.org/x/crypto v0.0.0-20191002192127-34f69633bfdc
+	github.com/u-root/u-root v6.0.1-0.20200822172051-c6181ee270c5+incompatible
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	google.golang.org/grpc v1.30.0 // indirect
 	harvey-os.org v0.0.0-20200727215830-5b0d35efdc89 // indirect
```
Comment (fhs) [inline go.mod:11]: This version of `harvey-os.org` differs from the one in `bootstrap.sh`.
```suggestion
	harvey-os.org v0.0.0-20200818173457-bcfa7228c80c // indirect
```

**Review by sevki — APPROVED** [review]
all Passed. LGTM

**Review by gmacd — APPROVED** [review]

**Pull request merged**